packages/build/src/homebrew/generate-formula.ts

```diff
@@ -33,6 +33,8 @@ export async function generateUpdatedFormula(
   if (
     currentVersion &&
     semver.compare(currentVersion, context.version, {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore 'includePrerelease' does not exist in type 'Options'.
```

Comment:
So.. I move the eslint config to a package and then suddenly this ts compile error appears.